neqo-transport/src/packet.rs

```diff
@@ -156,6 +157,44 @@ impl PacketHdr {
     pub fn body_len(&self) -> usize {
         self.body_len
     }
+
+    // header length plus auth tag
+    pub fn overhead(&self, aead: &Aead, pmtu: usize) -> usize {
+        match &self.tipe {
+            PacketType::Short => {
+                let mut enc = Encoder::default();
+                // Leading byte.
+                let pnl = pn_length(self.pn);
+                enc.encode_byte(PACKET_BIT_SHORT | PACKET_BIT_FIXED_QUIC | encode_pnl(pnl));
```

Comment:
Do we have to do the encoding here, or can we instead use `Encoder::varint_len` as necessary?  I'd rather not perform the allocations for the encoder unless we intend to use the output.

Reply:
ok cool.